src/lib/client.js

```diff
@@ -80,14 +81,16 @@ class Client extends EventEmitter {
 
   disconnect () {
     this.connecting = false
-    this.plugin.disconnect()
+    return this.plugin.disconnect()
   }
 
   /**
    * Get a quote from a connector
    * @param  {String} [params.sourceAmount] Either the sourceAmount or destinationAmount must be specified
    * @param  {String} [params.destinationAmount] Either the sourceAmount or destinationAmount must be specified
-   * @param  {String} params.destinationLedger Recipient's ledger
+   * @param  {String} params.destinationAddress Recipient's ledger
+   * @param  {String} [params.destinationPrecision] Must be provided for ledgers that are not adjacent to the quoting connector.
```

Comment:
We only need the `destinationPrecision` if quoting by `sourceAmount`. If the `destinationAmount` is the input we don't need to round it.